src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -89,6 +89,36 @@ def set_top_tags(
         logger.exception("We failed to set a tag.")
 
 
+def update_result_and_tags(
+    scope: Scope,
+    config_and_outcome: JSONData,
+    result: JSONData,
+) -> Dict[str, str]:
+    new_result_values = {"config": config_and_outcome["config"]}
+    found: bool = result["sourceUrl"] is not None
+    scope.set_tag("stacktrace_link.found", found)
+    scope.set_tag("stacktrace_link.empty_root", config_and_outcome["config"]["stackRoot"] == "")
+    scope.set_tag(
+        "stacktrace_link.auto_derived",
+        config_and_outcome["config"]["automaticallyGenerated"] is True,
```

Comment:
This tag is now calculated by using the serialized object of the code mapping. See change in `src/sentry/api/serializers/models/repository_project_path_config.py`.

Reply:
So we don't need to check the stack root for `automaticallyGenerated` anymore? (We previously had `filepath.startswith(config.stack_root)` as part of the condition for `derived = True`)

Reply:
Thanks for checking.

This will now grab the value from the config that matched (derived or not) OR the last one that was processed.